### PR TITLE
Murcake/update cairo lang to 0.6.2

### PR DESCRIPTION
**Pull request by murcake**

### CODE REVIEW

**Comment by swapnilraj** [comment]
felt_and and other felt bitwise operations need to use bitwise builtin now

**Comment by murcake** [comment]
> felt_and and other felt bitwise operations need to use bitwise builtin now

Yep, see the update!

**Review by BitBaseBit — APPROVED** [review]

File: warp/cairo-src/evm/uint256.cairo [hunk 1]
```diff
@@ -110,7 +112,8 @@ func uint256_signextend{range_check_ptr}(a : Uint256, i : Uint256) -> (res : Uin
     return (a)
 end
 
-func uint256_byte{range_check_ptr}(a : Uint256, i : Uint256) -> (res : Uint256):
+func uint256_byte{range_check_ptr, bitwise_ptr : BitwiseBuiltin*}(a : Uint256, i : Uint256) -> (
+        res : Uint256):
     let (i, _) = uint256_mul(i, cast((8, 0), Uint256))
```
Comment (cicr99) [inline warp/cairo-src/evm/uint256.cairo:117]: In this method you are still using `felt_and` function (line 120)

**Review by cicr99 — CHANGES_REQUESTED** [review]

File: warp/cairo-src/evm/uint256.cairo (hunk 1 above)
Comment (murcake) [reply warp/cairo-src/evm/uint256.cairo:117]: Yeah, I rolled back some changes and accidently put that back there. I'll fix it, but I think that #136 is more urgent

**Review by cicr99 — APPROVED** [review]

**Pull request merged**